src/java.desktop/share/classes/javax/swing/plaf/basic/BasicTextAreaUI.java

```diff
@@ -138,7 +138,8 @@ public Dimension getMinimumSize(JComponent c) {
 
     /**
      * Creates the view for an element.  Returns a WrappedPlainView or
-     * PlainView.
+     * PlainView if non-bidi text is handled
+     * else returns PlainParagraph or BoxView if bidi text is being handled.
      *
```

Comment:
So .. what if the text wasn't bidi but then some one adds bidi text ?

And can we change the summary of the BUG and PR to something grammatically correct, and perhaps even
descriptive such as
"BasicTextAreaUI:create incompletely documents the possible returned View types"

Reply:
In that case BasicTextUI#insertUpdate() will change the model which will again call BasicTextAreaUI#create to change the view to i18n to either of BoxView or PlainParagraph.